lib/smb2/packet/tree_connect_request.rb

```diff
@@ -4,13 +4,18 @@ class Smb2::Packet
   # [Section 2.2.9 SMB2 TREE_CONNECT Request](https://msdn.microsoft.com/en-us/library/cc246567.aspx)
   class TreeConnectRequest < Smb2::Packet
     nest :header, RequestHeader
-    unsigned :struct_size, 16
+    unsigned :struct_size, 16, default: 9
     # These two bytes are used in the response, but just padding in the
     # request
-    unsigned :unused, 16
+    unsigned :reserved, 16, default: 0
     data_buffer :tree
 
     rest :buffer
 
+    # @return [Symbol] a key in {Smb2::COMMANDS}
+    def self.command
```

Comment:
Change to `COMMAND = TREE_CONNECT` if you move to [class-constant lookup](https://github.com/rapid7/smb2/pull/28/files#r28820364)